Sources/Purchasing/EntitlementInfo.swift

```diff
@@ -136,7 +136,7 @@ extension PeriodType: DefaultValueProvider {
     /**
      The product plan identifier that unlocked this entitlement (usually for a Google Play purchase)
```

Comment:
Should we remove the word "usually" here?

Reply:
I pushed the change but it's not appearing in Github 😞. Will wait for a bit to see if it appears